node/node_test.go

```diff
@@ -123,65 +122,72 @@ func TestNewNode_ApplyNilOptionShouldError(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
-func TestGetBalance_NoAddrConverterShouldError(t *testing.T) {
+// Question for review: is this test still relevant (can coreComponents be only partly initialized)?
```

Comment:
no, we have unit tests that ensure the components are build up correctly (with all fields not-nil)

Reply:
Removed.